### ISSUE TITLE
ProgressBarAndroid Component does not announce "disabled"

### ISSUE THREAD

**Issue by fabriziobertoglio1987** [fabriziobertoglio1987/react-native-notes#3]
https://github.com/facebook/react-native/issues/30941 and https://github.com/react-native-progress-view/progress-bar-android/issues/62

### Description
>When using a screen reader the ProgressBarAndroid component does not announce that it is disabled.
>This issue is covered in-depth by parent issue https://github.com/facebook/react-native/issues/30840

**Comment by fabriziobertoglio1987** [comment]
### Other Accessibility Issues in ProgressBar

Related issue https://github.com/react-native-progress-view/progress-bar-android/issues/61#issuecomment-785188227

- ProgressBarAndroid is not focusable on [Android API 28][11]. The ScreenReader will not focus on the ProgressBar on API28, but works in [API30][12]. ScreenReader announces the values if [setFocusable][10] ([see e0db4aa][13] and [VIDEO][14]) is called on ProgressBar instance on API 28.

[10]: https://drive.google.com/file/d/1IN1sQc5kQ2bKoAJb11MFpMlQjqOJgwM4/view?usp=sharing "QuickFix to make ProgressBar focusable on API28"
[11]: https://user-images.githubusercontent.com/24992535/151913000-9d994844-12fa-4261-8290-775c8326ef8a.mp4 "API 28 ProgressBar not focusable from ScreenReader"
[12]: https://user-images.githubusercontent.com/24992535/151914592-235283a6-e116-4e74-90a1-32a1fed229b7.mp4 "API 30 Focusable Progress Bar"
[13]: https://github.com/fabriziobertoglio1987/progress-bar-android/commit/e0db4aad7106e64705f3f352e7014eda8f758f06 "commit invoking setFocusable on API 28"
[14]: https://user-images.githubusercontent.com/24992535/151916596-c94ddeea-f4b8-4781-8614-6fb3ff1a4197.mp4 "after calling setFocusable(true)"

- Debugging [setItemCount][20] ([IMAGE][22]), View method onFocus and logging with Android Studio ([VIDEO][21] and [more info][23]).

[20]: https://github.com/aosp-mirror/platform_frameworks_base/blob/1ac46f932ef88a8f96d652580d8105e361ffc842/core/java/android/widget/ProgressBar.java#L2328-L2329 "setItemCount"
[21]: https://drive.google.com/file/d/1A_lAe2A4NaUQ2Qa_UVd3umACJ3SaKRfA/view?usp=sharing "ProgressBar#setItemCount image"
[22]: https://drive.google.com/file/d/124sSBTCLCOI0zYXavKNF1hdqORAo6JKn/view?usp=sharing "Debugging Video"
[23]: https://docs.google.com/spreadsheets/d/1huhr26j7S4af26xInzdo3OWL1aAUS_qLpAx_BqzL7hY/edit#gid=114698190&range=E22:E24 "Investigating different accessibility methods in January"

**Comment by fabriziobertoglio1987** [comment]
Accessibility Issue on API 28

Commit https://github.com/fabriziobertoglio1987/progress-bar-android/commit/a6b7cc966277ae15a744b0b4df08f860a24eab1a adds the accessible prop from the [ReactViewManager][10] ([video][https://user-images.githubusercontent.com/24992535/151934126-fbce62f8-3a5a-4542-9749-ac98859796fb.mp4]).

[10]: https://github.com/fabriziobertoglio1987/react-native/blob/76a2cf3569571b943b4bcc6867e069338ff88f1f/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java#L57-L59 "ReactViewManager accessible prop"

Relevant code for adding accessibilityState: https://github.com/fabriziobertoglio1987/react-native/blob/76a2cf3569571b943b4bcc6867e069338ff88f1f/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L180

**Comment by fabriziobertoglio1987** [comment]
https://github.com/fabriziobertoglio1987/react-native-notes/issues/3 may fix this issue

**Comment by fabOnReact** [comment]
https://github.com/fabriziobertoglio1987/progress-bar-android/blob/a6b7cc966277ae15a744b0b4df08f860a24eab1a/android/src/main/java/com/reactnativecommunity/androidprogressbar/ReactProgressBarViewManager.java#L105-L108

The method is never called

```java
  @Override
  public void updateExtraData(ProgressBarContainerView root, Object extraData) {
    // do nothing
  }
```
and overrides

https://github.com/fabriziobertoglio1987/react-native-notes/blob/9cd43340a7e2443564c2ff5e8e85d37f6e1e47ef/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java#L331-L335

Seems not relevant for accessibility.

**Comment by fabOnReact** [comment]
- [ ] Check for other PRs fixing similar issue
